src/test/java/org/gridsuite/loadflow/server/LoadFlowControllerTest.java

```diff
@@ -792,4 +790,53 @@ private static final class LimitViolationsMock {
                 new LimitViolation("NHV1_NHV2_2", "lineName2", LimitViolationType.CURRENT, "limit2", 300, 900, 0.7F, 1000, TwoSides.ONE),
                 new LimitViolation("NHV1_NHV2_2", "lineName2", LimitViolationType.CURRENT, "limit2", 300, 900, 0.7F, 1000, TwoSides.TWO));
     }
+
+    private static final class VoltageLimitViolationsMock {
```

Comment:
Why this static block ?

Reply:
i need results with  ViolationLocation for the tests